.github/workflows/balena.yml

```diff
@@ -575,8 +575,7 @@ jobs:
 
               ssh-uuid -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null \
               ${{ steps.register-test-device.outputs.balena_device_uuid }}.balena \
-              'balena ps -q | xargs balena inspect | jq -r .[].State.Status' \
-                | grep -E 'created|restarting|removing|paused|exited|dead' 
```

Comment:
What's the reason for this change? I'm assuming this changes the output seen in CI stdout used for debugging/logging purposes?

Reply:
Yes, it's a "verbose mode" to help debugging if the github workflow hangs there